cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/CassandraScheduler.java

```diff
@@ -141,6 +142,12 @@ public CassandraScheduler(
     public void registered(SchedulerDriver driver,
                            Protos.FrameworkID frameworkId,
                            Protos.MasterInfo masterInfo) {
+        // Return early if registered() has already been called once in the lifetime of this object.
+        if (!isSchedulerRegistered.compareAndSet(false, true)) {
```

Comment:
The reconciler should already be started on L173 and reconcile() is called every time we receive an offer. Do you want me to call reconciler.start() again? Or reconcile() ?